<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsSingleBandPseudoColorRendererWidgetBase</class>
 <widget class="QWidget" name="QgsSingleBandPseudoColorRendererWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="1">
      <widget class="QLineEdit" name="mMinLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="mMinLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Min</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="mBandLabel">
       <property name="text">
        <string>Band</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="mMaxLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Max</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="mMaxLineEdit"/>
     </item>
     <item row="0" column="1" colspan="3">
      <widget class="QgsRasterBandComboBox" name="mBandComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="mMinMaxContainerWidget" native="true"/>
   </item>
   <item>
    <widget class="QgsColorRampShaderWidget" name="mColorRampShaderWidget" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsRasterBandComboBox</class>
   <extends>QComboBox</extends>
   <header>raster/qgsrasterbandcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsColorRampShaderWidget</class>
   <extends>QWidget</extends>
   <header>raster/qgscolorrampshaderwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
